tools/release/setup_version.py

```diff
@@ -119,6 +121,7 @@ def __setup_prod_version(version: Version, target_version: str, branch_name: str
 
         if _environment == "dev":
             __setup_dev_version(_version, _path, _name)
+            bump_ext_version(_version, _path)
```

Comment:
It's not clear to me. Here is my understanding of the flow: 

- We dev release using the version in the version.json files.
- We bump up the version and write the version.json files
- We commit and push the changes

If this is correct, I don't get why you changed the version before releasing.